JASP-Engine/JASP/R/commonMachineLearningClassification.R

```diff
@@ -136,16 +137,21 @@
   
   }
   
+  # Add common columns
   classificationTable$addColumnInfo(name = 'ntrain', title = 'n(Train)', type = 'integer')
-  classificationTable$addColumnInfo(name = 'nvalid', title = 'n(Validation)', type = 'integer')
+  if(options[["modelOpt"]] != "optimizationManual")
```

Comment:
there are two identical if statements, maybe
```r
if(options[["modelOpt"]] != "optimizationManual") {
  ...
}
```

Reply:
The order of the columns is set by this part right? All set sizes should be placed together, and all accuracy scores as well. That's why I did this, or can it be done in some other way?

Reply:
@koenderks 's solution seems the right one here.